falcon/request.py

```diff
@@ -293,6 +295,10 @@ def client_accepts_json(self):
         return self.client_accepts('application/json')
 
     @property
+    def client_accepts_jsonapi(self):
```

Comment:
That's fine & I can understand it. I just passed in my own request object & that has plenty of flexibibility for what I need.